File: src/src/controller/menuController.py
import requests
from src.setup import SERVER_URL
import json


def getMenu(token: str):
    # some code
    response = requests.get(
        f"{SERVER_URL}/kitchen/menu",
        headers={"Authorization": f"Bearer {token}"},
    )
    if response.ok:
        return response.json()
    print(response.json())
    return None

File: frontend/src/controllers/menuController.js
import { SERVER_URL } from "../setup";

const getMenu = async (token) => {
  try {
    const res = await fetch(SERVER_URL + "/kitchen/menu", {
      headers: {
        Authorization: `Bearer ${token}`,
      },
    });
    const data = await res.json();
    if (res.ok) {
      data.forEach((element) => {
        element.selected = false;
      });
      return data;
    } else {
    }
  } catch (err) {
    console.log(err);
  }
};

export { getMenu };
